pkgs/applications/editors/lite-xl/default.nix

```diff
@@ -16,30 +16,18 @@
 
 stdenv.mkDerivation rec {
   pname = "lite-xl";
-  version = "2.0.1";
+  version = "2.0.3";
 
   src = fetchFromGitHub {
     owner = "lite-xl";
     repo = "lite-xl";
     rev = "v${version}";
-    sha256 = "sha256-+RbmT6H/5Ldhv3qOClxMjCSGMudbkGtkjo2SpGqExao=";
+    sha256 = "sha256-8Hw2zDYynWG/NwxtWgbEVUCzrtRVkOhwtsisURNqUn8=";
   };
 
   patches = [
-    # Fixes compatibility with Lua5.2, remove patch when PR merged
-    # https://github.com/lite-xl/lite-xl/pull/435
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
You mean the one that I replaced with the .patch file?
It was merged but not included in the v2.0.3 release, so we still need this to the package to work. Pretty sure we can remove this on v2.0.4, though.

Reply:
remove the fetchpatch in the input arguments line 3

Reply:
Ah, didn't notice that. My bad.

Reply:
> It was merged but not included in the v2.0.3 release, so we still need this to the package to work.

Why not keep fetching it with `fetchpatch` then? I think we generally prefer not having in-tree patches if they can be fetched from somewhere else or upstreamed.

Reply:
Patch seems to fail to automatically apply on 2.0.3, so thats why I made some changes.